modules/log6x-6-1-5-rn.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.1/log6x-release-notes-6.1.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-6-1-5_{context}"]
+= Logging 6.1.5 Release Notes
+
+This release includes link:https://access.redhat.com/errata/RHSA-2025:3907[RHSA-2025:3907].
+
+[id="logging-release-notes-6-1-5-enhancements_{context}"]
+== New features and enhancements
+
+* Before this update, time-based stream sharding was not enabled in Loki, which resulted in Loki being unable to save historical data. With this update, {loki-op} enables time-based stream sharding in Loki, which helps Loki save historical data. (link:https://issues.redhat.com/browse/LOG-6991[LOG-6991])
+
+
+
+
```

Comment:
Just a nitpick, but I think I'd remove the three extra empty lines.

Reply:
fixed!